.rspec

```diff
@@ -1,3 +1,2 @@
 --color
 --require spec_helper
---backtrace
```

Comment:
Yes, we could do that.

In theory you should be able to add it to your global .rspec file - https://relishapp.com/rspec/rspec-core/v/2-0/docs/configuration/read-command-line-configuration-options-from-files

Out of curiosity, how come you find it useful all the time? I've found it useful very few times